maps/creatives/simid_maps.html

```diff
@@ -0,0 +1,9 @@
+<html>
+  <head>
+    <script src="../simid_protocol.js"> </script>
```

Comment:
I think you can move the simid_protocol class into this folder.
It was a folder above in the SIMID repo because its being used by the player code as well, but since there is no player here, its only being used by the creatives and can go into that folder

Reply:
Is this true even though we are using the same player?

Reply:
Yeah cause you won't be copying over any of the player code into this repo.  You can use the player being hosted by the simid repo, but no need to keep this file in a shared location, since it isn't being shared in this repo but is only being used by creatives

Reply:
okay great fixed